app/view/account_transaction_details.py

```diff
@@ -128,7 +149,7 @@ def put(self, id):
         """Update the account transaction detail """
         try:
             data = request.get_json()
-            result = account_transaction_detail_schema.validate(data)
+            result = account_transaction_detail_schema.validate(data, partial=True)
```

Comment:
What is impact of `partial=True`?

Reply:
partial=True is used because we don't have to fill the data of every field while update it have given the flexibility that only one field can also be updated.

Reply:
partial=True is used because we don't have to fill the data of every field while update it have given the flexibility that only one field can also be updated.